src/components/PhishingWarningBanner/index.tsx

```diff
@@ -45,10 +44,14 @@ const PhishingWarningBanner: React.FC = () => {
   const [, hideBanner] = usePhishingBannerManager()
   const { isMobile, isMd } = useMatchBreakpoints()
   const warningText = t("please make sure you're visiting https://pancakeswap.finance - check the URL carefully.")
-  const warningTextAsParts = warningText.split(/(https:\/\/pancakeswap.finance)/g)
+  const warningTextAsParts = warningText
+    .split(/(https:\/\/pancakeswap.finance)/g)
+    .flatMap((text) => text.split(/(https:\/\/)/g))
+    .filter((n) => n)
```

Comment:
What is this filter doing? Filtering out empty strings? Are there any? 🤔

Reply:
Yes list comes like that 

["please make sure you're visiting ", "", "https://", "pancakeswap.finance", " - check the URL carefully."]